### DISCUSSION TITLE
Imported deposits have Source Account = (no name)?

### DISCUSSION THREAD

**Discussion by jesta030** [firefly-iii/firefly-iii#6257] [Firefly III]
[Edit] This looks like a bug so I opened https://github.com/firefly-iii/firefly-iii/issues/6259

Thanks for the great software!

I am importing my banks CSV and the only way to differentiate between incoming and outgoing transactions is by the amount being smaller or greater zero. Otherwise there's two columns containing account numbers: one has my account and the other has the opposing account. All transactions no matter whether deposit or withdrawal have my account in the first column and the sender or recipient in the other column.

Firefly correctly categorises transactions as deposits or withdrawals based on this but deposits don't get a source account assigned, instead it just says (no name). I guess firefly is confused because the source account appears to be my own and just substitutes it with (no name)?

I tried looking into rules to maybe swap source and destination when amount is larger than zero but that doesn't seem to be an option.

Any hints how to fix this? If more info is needed I can fill it in later, in a hurry currently.

**Comment by JC5** [comment]
See also https://github.com/firefly-iii/firefly-iii/issues/6259
